### ISSUE TITLE
[a11y] Focus order not showing on table rows to open single record

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#279]
Reported issue:
You should be able navigate interactive elements using a keyboard via tabbing. The focus should be clearly indicated. On the CRT view, the focus order disappears when you get to the row:

Steps to reproduce:
- On the view all table, hit tab. You'll notice the focus order move through page elements (skip to main, logout, add new...etc)
- Once you get passed 'incident location' your focus will disappear
- If you hit 'enter' on your keyboard it will open a complaint record page, indicating that some focus was somewhere but not showing

Expected behavior:
- I would expect the row to highlight with the same hover drop shadow, but I understand it's probably easier for each cell to get a the blue focus, which is fine.

**Comment by Jacklynn** [comment]
🕺
